department-of-veterans-affairs/va.gov-cms
# VACMS-4935: Update IEF for Vet Center health services.

## Description

See #4935. 

## Testing done

Behat, visual.

## Screenshots


![Edit_Vet_Center_Escanaba_Vet_Center___VA_gov_CMS](https://user-images.githubusercontent.com/643678/113646707-5a863c80-9657-11eb-9426-df58427dbbf5.png)


## QA steps

* As a sue.candelier@va.gov
* Edit node/3756/edit 
* Edit, then add a servivce
- [x]  You should be able to set the Owner value (which was previously not there).



### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
